### PR TITLE
Feat: Expose descriptions for currently-bound hotkeys.

### PR DESCRIPTION
**Pull request by zenzen-sol**
This PR exposes the `description` option in the `hotkeys` array returned by `useHotkeysContext()`.  Helpful if you want to display a list of currently-active hotkeys to the user.

### CODE REVIEW

**Comment by JohannesKlauss** [comment]
Please add a test that tests the behavior, other than that it looks good!

**Comment by OneStepCloser** [comment]
I was about to embed the library into our project and I must say, that this feature of providing description would be really helpful in our case. Looking forward to this MR to be merged. 🙏

**Comment by zenzen-sol** [comment]
Sure. I'll add a test ASAP.

File: tests/HotkeysProvider.test.tsx [hunk 1]
```diff
@@ -220,4 +239,21 @@ test('should return bound hotkeys when defined as a string array', () => {
   })
   expect(result.current.hotkeys[0].keys).toEqual(['a', 'c'])
   expect(result.current.hotkeys[1].keys).toEqual(['b'])
-})
\ No newline at end of file
+})
+
+test('should return descriptions for bound hotkeys', () => {
```
Comment (JohannesKlauss) [inline tests/HotkeysProvider.test.tsx:244]: The same test appears in Line 181

**Review by JohannesKlauss — CHANGES_REQUESTED** [review]

File: tests/HotkeysProvider.test.tsx (hunk 1 above)
Comment (zenzen-sol) [reply tests/HotkeysProvider.test.tsx:227]: Sorry...meant to move it, and left a copy. I also reverted a bad merge in `useHotkeys.ts` that broke this feature.

**Review by JohannesKlauss — APPROVED** [review]

**Pull request merged**